andela/ah-backend-tesseract
# #159951963 Email should be sent to a user upon successful registration for verification

#### What does this PR do?
- sends email to a user upon registration for verification
#### Description of Task to be completed?
- send activation email to user that has registered
- confirm token validity
- activate users account
- add tests for activation
#### How should this be manually tested?
- create add email specific settings in the .env file as specified in the .env.sample file 
- create an account using a valid email 
- attempt to login, this should fail because the account is unaccessible before activating
- check you email for the verification link
- click on the link to activate your account 
- attempt to login again. this should be successful

#### What are the relevant pivotal tracker stories?
[finishes #159951963]

Co-authored-by: Ruganda Mubarak <mubaruganda@gmail.com>
